buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -353,11 +353,11 @@ abstract class PublishingPlugin : Plugin<Project> {
           val releaseNotesTask = tasks.named("makeReleaseNotes")
 
           from(releaseNotesTask) { into("android/client/$directory/_releases") }
-          rename { "$releaseName.md" }
+          rename { "${releaseName.toLowerCase()}.md" }
         }
       }
 
-      destinationDir = project.fileFromBuildDir("release-notes")
+      destinationDir = project.fileFromBuildDir("release_notes")
```

Comment:
You'll also need to change [this](https://github.com/firebase/firebase-android-sdk/blob/7b940f822555f554318cd7df64d6a67537bdc3e2/buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt#L112) and [this](https://github.com/firebase/firebase-android-sdk/blob/7b940f822555f554318cd7df64d6a67537bdc3e2/buildSrc/src/test/kotlin/com/google/firebase/gradle/plugins/MakeReleaseNotesTests.kt#L34)